src/d/actor/d_a_dummy.cpp

```diff
@@ -4,39 +4,100 @@
 //
 
 #include "d/actor/d_a_dummy.h"
-#include "dolphin/types.h"
+#include "d/d_procname.h"
+#include "f_op/f_op_actor_mng.h"
+#include "m_Do/m_Do_mtx.h"
 
 /* 00000078-0000009C       .text solidHeapCB__Q27daDummy5Act_cFP10fopAc_ac_c */
-void daDummy::Act_c::solidHeapCB(fopAc_ac_c*) {
-    /* Nonmatching */
+BOOL daDummy::Act_c::solidHeapCB(fopAc_ac_c* i_this) {
+    return ((Act_c*)i_this)->create_heap();
 }
 
 /* 0000009C-000000A4       .text create_heap__Q27daDummy5Act_cFv */
-void daDummy::Act_c::create_heap() {
-    /* Nonmatching */
+bool daDummy::Act_c::create_heap() {
+    return 1;  
```

Comment:
```suggestion
    return true;  
```
true/false are preferred for bool